pilot/pkg/features/pilot.go

```diff
@@ -400,4 +400,7 @@ var (
 		15*time.Second,
 		"If set, the max amount of time to delay a push by. Depends on PILOT_ENABLE_FLOW_CONTROL.",
 	).Get()
+
+	EnableAuthzRegexMatching = env.RegisterBoolVar("AUTHZ_ENABLE_REGEX", false,
+		"If true, regex matching will be enabled for authorization policy. In the policy, regex expression after 'regex:' will be used").Get()
```

Comment:
`In the policy, regex expression after 'regex:' will be used` -> `In the policy, text after `regex:` will be interpreted as regular expression`